lib/lang/latin/latin-language-dataset.js

```diff
@@ -273,17 +273,29 @@ export default class LatinLanguageDataset extends LanguageDataset {
   // for Lemmas
   addVerbForms (partOfSpeech, data) {
     // First row are headers
+    this.verbsIrregularLemmas = []
+
     for (let i = 1; i < data.length; i++) {
       const item = data[i]
       let lemma = item[0]
       // let principalParts = item[1].split(/_/)
+      let principalParts = item[1]
```

Comment:
A few notes about the data:
the values in the principal parts field are separated in _ to avoid needing to quote that field in the csv, it's really a list of _ separated values and should be treated as such. that's what why I had before the split on _ line 281.  In our Lemma data model, there is a principalParts field, this is where these values would be found in the lemma of the homonym that would be matched against the table data. I'm not sure we *need* to match on the hdwd and the principal parts but we do want to represent this consistently. So I think we might actually want to instantiate a Lemma object with the hdwd and principal parts from the inflection data.